api/lib.go

```diff
@@ -50,19 +55,19 @@ func ReleaseCache(cache Cache) {
 }
 
 func Create(cache Cache, wasm []byte) ([]byte, error) {
-	code := sendSlice(wasm)
-	defer freeAfterSend(code)
+	w := makeView(wasm)
+	defer runtime.KeepAlive(wasm)
```

Comment:
Same comments about moving KeepAlive out of the defer block. Otherwise good stuff
(and all other examples)